web/src/lib/stores/assets.store.ts

```diff
@@ -153,15 +153,15 @@ export class AssetStore {
     this.emit(true);
   }, 10_000);
 
-  async init(viewport: Viewport) {
+  async init(viewport: Viewport, ascendingOrder?: boolean) {
```

Comment:
This is the wrong place to pass the option. You should pass it when the class is created like everything else.